node/lib/cmd/open.js

```diff
@@ -156,7 +156,8 @@ exports.executeableSubcommand = co.wrap(function *(args) {
     let subsToOpen = yield SubmoduleUtil.resolveSubmoduleNames(workdir,
                                                                cwd,
                                                                subs,
-                                                               paths);
+                                                               paths,
+                                                               true);
```

Comment:
to me this is a bit hacky. IMO, it is better to return full result, like a map of sub names to actually opened codebases, and caller can then decide whether to and throw exception if need to. But I can imagine that will introduce plenty of changes.